packages/shorebird_cli/lib/src/commands/build/build_aar_command.dart

```diff
@@ -79,7 +76,7 @@ class BuildAarCommand extends ShorebirdCommand
       'host',
       'outputs',
       'repo',
-      ...androidPackageName.split('.'),
+      ...androidPackageName!.split('.'),
```

Comment:
why do we need to force unwrap here?

Reply:
It's a computed property defined in `ShorebirdConfigMixin`. We check if it's null above, but because it isn't a local variable, the compiler doesn't let us treat it as non-null.

Reply:
ah gotcha missed that it was moved to the mixin